[fix] 중복된 컬럼 매핑 문제

## 🤔 오류 내용
Caused by: org.hibernate.MappingException: Column 'member_id' is duplicated in mapping for entity 'team9.ddang.member.entity.Friend' (use '@Column(insertable=false, updatable=false)' when mapping multiple properties to the same column)
<br>


## ⚠ 에러 캡쳐 

<br>

--- a/src/main/java/team9/ddang/member/entity/Friend.java
+++ b/src/main/java/team9/ddang/member/entity/Friend.java
@@ -18,11 +18,11 @@ public class Friend extends BaseEntity {
     private Long friendId;
 
     @ManyToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "member_id", nullable = false)
+    @JoinColumn(name = "sender_id", nullable = false)
     private Member senderId;
 
     @ManyToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "member_id", nullable = false)
+    @JoinColumn(name = "receiver_id", nullable = false)
     private Member receiverId;
 
     @Builder

--- a/src/main/java/team9/ddang/member/entity/FriendRequest.java
+++ b/src/main/java/team9/ddang/member/entity/FriendRequest.java
@@ -17,11 +17,11 @@ public class FriendRequest extends BaseEntity {
     private Long friendRequestId;
 
     @ManyToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "member_id", nullable = false)
+    @JoinColumn(name = "sender_id", nullable = false)
     private Member senderId;
 
     @ManyToOne(fetch = FetchType.LAZY)
-    @JoinColumn(name = "member_id", nullable = false)
+    @JoinColumn(name = "receiver_id", nullable = false)
     private Member receiverId;
 
     @Builder

